From adf4b5c5d136bdbc1ed1cf028d4a74795447d879 Mon Sep 17 00:00:00 2001
From: Erwann Roussy <erwann.roussy@savoirfairelinux.com>
Date: Wed, 10 Dec 2025 17:54:16 +0100
Subject: [PATCH] pyproject.toml: rollback to old licence field

This licence field is too new for the setuptools used in scarthgap.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2b96e83..7e0c4c9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@
 name = "vm_manager"
 version = "0.1"
 description = "Managed VMs in Seapath cluster"
-license = "Apache-2.0"
+license = { text = "Apache-2.0" }
 authors = [
 	{name = "RTE"},
 	{name = "Mathieu Dupré", email = "mathieu.dupre@savoirfairelinux.com"},
-- 
2.52.0

